README.md

```diff
@@ -111,7 +111,7 @@ public function boot()
 This package is installed via [Composer](https://getcomposer.org/). To install, run the following command.
 
 ```bash
-composer require dyrynda/laravel-model-uuid
+composer require "dyrynda/laravel-model-uuid:~1.0"
```

Comment:
I installed this today on the current version of Laravel along with Michael's other uuid package and it was borked out-of-the-box. Why the ~1.0 constraint? Installing 2.0.2 corrected the errors.